src/app/components/import/ImportModal.js

```diff
@@ -53,12 +54,13 @@ const ImportModal = ({ onClose, ...rest }) => {
     };
 
     const handleAttach = ({ target }) => {
-        const attachedFile = [...target.files].filter(({ type }) => ['text/vcard', 'text/csv'].includes(type))[0];
+        const ext = splitExtension(target.files[0].name)[1];
```

Comment:
```suggestion
        const [ , ext] = splitExtension(target.files[0].name);
```

Reply:
what about adding a function that just returns the extension of the file? Sounds more logic no?

Reply:
I'm using the function `splitExtension` in `components/import/AttachedFile` and I need the filename and the extension in there, so that's why it returns two elements. In general I think it can be useful to have the function behave as `split` and return an array. But if you think it's useful to create an additional `getExtension`, it can be done